### PR TITLE
make configurable a child of aws

### PR DESCRIPTION
**Pull request by kiiadi**
Screen Shot 2021-09-30 at 2 23 28 PM

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.form [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <properties/>
     <border type="none"/>
     <children>
-      <grid id="cb6a8" binding="serverlessSettings" layout-manager="GridLayoutManager" row-count="2" column-count="3" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
+      <grid id="cb6a8" binding="serverlessSettings" layout-manager="GridLayoutManager" row-count="1" column-count="3" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
```
Comment (manodnyab) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.form:11]: What change is this representing?

**Review by manodnyab — APPROVED** [review]

File: jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.form (hunk 1 above)
Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.form:11]: This is the removal of the Lambda gutter-icon preference which is the only one stored at the project level.

**Review by manodnyab — APPROVED** [review]

**Pull request merged**